bin/templates/cordova/lib/check_reqs.js

```diff
@@ -393,11 +393,13 @@ module.exports.run = function () {
         console.log('ANDROID_SDK_ROOT=' + process.env['ANDROID_SDK_ROOT'] + ' (recommended setting)');
         console.log('ANDROID_HOME=' + process.env['ANDROID_HOME'] + ' (DEPRECATED)');
 
-        if (!String(values[0]).startsWith('1.8.')) {
+        if (values[0] === undefined) {
             throw new CordovaError(
-                'Requirements check failed for JDK 8 (\'1.8.*\')! Detected version: ' + values[0] + '\n' +
-                'Check your ANDROID_SDK_ROOT / JAVA_HOME / PATH environment variables.'
+                'Requirements check for Java Development Kit failed. Detected version: ' + values[0] + '\n' +
+                'Check your Java installation and ANDROID_SDK_ROOT / JAVA_HOME / PATH environment variables.'
             );
+        } else {
+            console.log('Detected JDK version: ' + values[0] + '\n');
```

Comment:
I'm not sure if we want to add this output. If so, it needs to use `events` like everything else.

Reply:
I would prefer this text to be `verbose` output, personally.